mitosheet/mitosheet/step_performers/column_headers_transform.py

```diff
@@ -63,7 +64,16 @@ def execute(cls, prev_state: State, params: Dict[str, Any]) -> Tuple[State, Opti
                     new_columns.append(col)
         else:
             raise ValueError(f'Unknown transformation type: {transformation["type"]}')
-        
+
+        if len(set(new_columns)) < len(new_columns):
```

Comment:
@aarondr77 why don't you just use the new deduplicate functionality here? We could just have it option in this step, since it's likely so rare.